.github/workflows/build_and_publish_images.yml

```diff
@@ -0,0 +1,65 @@
+name: Build and Publish Base Images
+
+on:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: "0 5 * * Sun"
+
+jobs:
+  build-publish-image:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: './images'
+    steps:
+      - name: 'Checkout GitHub Action'
+        uses: actions/checkout@v4
+
+      - name: 'Login to GitHub Container Registry'
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{github.actor}}
+          password: ${{secrets.GITHUB_TOKEN}}
+
+      - name: 'Build and Publish alpine image'
+        run: |
+          docker build -t ghcr.io/cfpb/regtech/sbl/alpine:3.18_${{github.run_attempt}} -f Dockerfile-alpine .
+          docker push ghcr.io/cfpb/regtech/sbl/alpine:3.18_${{github.run_attempt}}
+
+          docker build -t ghcr.io/cfpb/regtech/sbl/alpine:v3.18.0 -f Dockerfile-alpine .
+          docker push ghcr.io/cfpb/regtech/sbl/alpine:v3.18.0
```

Comment:
we don't need to build twice do we? think we can just build once, and tag and push twice. also, not sure if this has already been discussed but since the convention is having `:latest` as the actual latest tag, what do we think about it being something like `ghcr.io/cfpb/.../alpine_3_18_0:latest` and `ghcr....alpine_3_18_0:{{github.run_attempt}}`?

Reply:
Yup. Thanks for catching that, forgot to remove the redundant build. I'll remove that.
As far as using the latest moniker, I hadn't heard anything about using `latest` tag. I was assuming that wasn't desired. 
However it might be moot because to make packages immutable in GHCR, we have to use semantic versioning which is why I changed the standard tag to that and just added the build specific one so people can pin if they need to.
Immutable packages is not 100% supported yet as it's still in Preview mode, but it does require semantic versioning from what I can tell. I put some references in the `contributing.md` in this PR.

Reply:
updated with both latest and semantic version in addition to a build specific tag. Doesn't hurt to have both ;-)